src/lang/std/engineConnection.ts

```diff
@@ -314,223 +611,184 @@ class EngineConnection {
       }
 
       let resp = message.resp
-      if (!resp) {
-        // If there's no body to the response, we can bail here.
+
+      // If there's no body to the response, we can bail here.
+      // !resp.type is usually "pong" response for our "ping"
+      if (!resp || !resp.type) {
         return
       }
 
-      if (resp.type === 'sdp_answer') {
-        let answer = resp.data?.answer
-        if (!answer || answer.type === 'unspecified') {
-          return
-        }
+      console.log('received', resp)
```

Comment:
we ought to clean this up soon, this is likely going to be wicked noisy

Reply:
This is only for websocket requests - from my experience, it's not that noisy - easily removed but should we maybe consider that it could be useful in the short term for when people are reporting bugs to us?

Reply:
I'm not worried about removing it for this PR @lf94 - let's leave it as is, land this and then remove it when/if it's an issue, I think it seems real helpful at the moment

Reply:
:ok_man: